release-tools/scripts/version-info.sh

```diff
@@ -0,0 +1,69 @@
+#!/bin/bash
+
+###### Information ############################################################################
+# Name:          version-info.sh
+# Maintainer:    ODFE Infra Team
+# Language:      Shell
+#
+# About:         Print the ES/ODFE related information and values
+#                as defined in the 'manifest.yml' file
+#
+# Usage:         ./version-info.sh $VERSION_TYPE
+#                $VERSION_TYPE (required): --od | --od-prev | --od-next
+#                                          --es | --es-prev | --es-next
+#
+# Requirements:  Need to install yq (v4.0.0+) on your system:
+#                https://github.com/mikefarah/yq/releases/
+#
+# Starting Date: 2021-01-06
+# Modified Date: 2021-01-06
+###############################################################################################
+
+set -e
+ROOT=`dirname $(realpath $0)`;
+PLUGIN_LIST="$ROOT/manifest.yml"
```

Comment:
Can you call it rename variable to `manifest`. Plugin list is confusing in the code

Reply:
That is due to old file name is plugins.json, will change it.

Reply:
fixed.

Reply:
Not fixed here